Sorting/SortingUtils.h

```diff
@@ -9,15 +9,24 @@
 
 #ifndef VECTOR_H
 #include <vector>
-#endif
+#include <cstdlib>
 
+#endif
+#define EXIT_CODE_ERROR -1
```

Comment:
Hi! First, let me thank you for the review, it's the first time I do a pull request, so I appreciate the feedback. I love the idea with your project, and I forked it because I wanted to improve my basics knowledge on C++ (more basic than I remembered as I see :( ) but, thank you, I will fix this, and I'm happy to help in other issues.